docs/source/writing_benchmarks.rst

```diff
@@ -59,8 +59,8 @@ that your benchmarks may produce, pass the ``--show-stderr`` flag::
 
        asv run --show-stderr
 
-Finally, there is a special command, ``asv dev``, that uses all of
-these features and is equivalent to::
+Finally, a quick way to test out the benchmark suite before doing a full run is to use all of
+these features together with::
```

Comment:
Maybe a [versionchanged directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionchanged) here to mention that there used to be `asv dev` for the same?

Reply:
That's up to you.

Personally I prefer docs to refer to the latest version of things; since the argument was removed, there must have been a reason why, so why bother educating users about a feature that is now unsupported?

Likewise, if they are intentionally downpinning to a past version, the past version of the docs at that point in time would still be valid in that respect

Reply:
For upgrade ease typically, if users come by trying to migrate from a workflow with `asv dev` it would make sense to highlight the alternative. I'll just push a minor update and get this through :)